thefuck/rules/pip_install_sudo.py

```diff
@@ -0,0 +1,13 @@
+import re
+from thefuck.utils import sudo_support
+
+
+def match(command, settings):
+    return (('pip' in command.script and 'install' in command.script) and
+            'failed with error code 1' in command.stderr and
+            ('Errno 13' in command.stdout or
```

Comment:
@ya0 FYI, Errno 13 means EACCES in <errno.h> and it's accurate error message in python code.

http://man7.org/linux/man-pages/man3/errno.3.html